<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tutorial</class>
 <widget class="QDialog" name="Tutorial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1600</width>
    <height>1200</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Tutorial {
border-image: url(:/images/resources/Other/TitleScreenBackground.png) 0 0 0 0 stretch stretch;
}</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <property name="spacing">
    <number>7</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="titleBar" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(84,94,106);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="titleBarSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>87</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="closeButton">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#closeButton {
color: white;
background-color: rgb(84,94,106);
border-style: solid;
}
#closeButton:hover {
background-color: red;
}</string>
        </property>
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <widget class="QWidget" name="widget_3" native="true">
       <property name="styleSheet">
        <string notr="true">#widget_3 {
background-color: rgb(40,60,69);
border-radius: 16px;
border-style: solid;
border-width: 4px;
border-color: rgb(11,29,41);
}
QLabel {
color: rgb(254,220,105);
}</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6" stretch="1,1,3">
        <item>
         <widget class="QLabel" name="introInfoLabel">
          <property name="styleSheet">
           <string notr="true">font-size: 18px;</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;FOOL'S GOLD&lt;/span&gt; is a free-to-play clicker game created by &lt;span style=&quot; font-weight:700;&quot;&gt;Adrian Suliga&lt;/span&gt; and published on &lt;span style=&quot; font-weight:700;&quot;&gt;GitHub&lt;/span&gt;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignJustify|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="goalLabel">
          <property name="styleSheet">
           <string notr="true">font-size: 16px;</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Your goal is to earn as much money as possible by mining valuable &lt;span style=&quot; font-weight:700;&quot;&gt;ores&lt;/span&gt; in the &lt;span style=&quot; font-weight:700;&quot;&gt;mine&lt;/span&gt;. Amount of money you get from a single ore is multiplied by number which is based on &lt;span style=&quot; font-weight:700;&quot;&gt;tools&lt;/span&gt; you bought from&lt;span style=&quot; font-weight:700;&quot;&gt; SHOP&lt;/span&gt;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignJustify|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="3,1">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="minimumSize">
                 <size>
                  <width>60</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">border-image: url(:/images/resources/Ores/selenite_ore.png) 0 0 0 0 stretch stretch;</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_6">
                <property name="minimumSize">
                 <size>
                  <width>60</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">border-image: url(:/images/resources/Ores/onyx_ore.png) 0 0 0 0 stretch stretch;</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_7">
                <property name="minimumSize">
                 <size>
                  <width>60</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">border-image: url(:/images/resources/Ores/amethyst_ore.png) 0 0 0 0 stretch stretch;</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_9">
                <property name="styleSheet">
                 <string notr="true">border-image: url(:/images/resources/Ores/diamond_ore.png) 0 0 0 0 stretch stretch;</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_10">
                <property name="styleSheet">
                 <string notr="true">border-image: url(:/images/resources/Ores/gold_ore.png) 0 0 0 0 stretch stretch;</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label_11">
                <property name="styleSheet">
                 <string notr="true">border-image: url(:/images/resources/Ores/jade_ore.png) 0 0 0 0 stretch stretch;</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_12">
                <property name="styleSheet">
                 <string notr="true">border-image: url(:/images/resources/Ores/ruby_ore.png) 0 0 0 0 stretch stretch;</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="soaLabel">
              <property name="styleSheet">
               <string notr="true">font-size: 16px;
</string>
              </property>
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt; have base worth equal to &lt;span style=&quot; font-weight:700;&quot;&gt;100 $&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="dgLabel">
              <property name="styleSheet">
               <string notr="true">font-size: 16px;</string>
              </property>
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt; have base worth equal to &lt;span style=&quot; font-weight:700;&quot;&gt;150 $&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="jrLabel">
              <property name="styleSheet">
               <string notr="true">font-size: 16px;</string>
              </property>
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;have base worth equal to&lt;span style=&quot; font-weight:700;&quot;&gt; 500 $&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,3">
       <item>
        <widget class="QLabel" name="titleLabel">
         <property name="styleSheet">
          <string notr="true">color: rgb(254,220,105);
font: 24pt;
background-color: rgb(40,60,69);
border-radius: 16px;
border-style: solid;
border-width: 4px;
border-color: rgb(11,29,41);</string>
         </property>
         <property name="text">
          <string>TUTORIAL</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="1,4">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <property name="styleSheet">
            <string notr="true">#widget_2 {
background-color: rgb(40,60,69);
border-radius: 16px;
border-style: solid;
border-width: 4px;
border-color: rgb(11,29,41);
}
QLabel {
color: rgb(254,220,105);
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3" stretch="2,1,1,1,1,1">
            <item>
             <widget class="QLabel" name="shopLabel">
              <property name="styleSheet">
               <string notr="true">#shopLabel {
padding: 6px;
font-size: 20px;
}</string>
              </property>
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Buying&lt;span style=&quot; font-weight:700;&quot;&gt; items&lt;/span&gt; from &lt;span style=&quot; font-weight:700;&quot;&gt;shop &lt;/span&gt;increases your&lt;span style=&quot; font-weight:700;&quot;&gt; multiplier&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignJustify|Qt::AlignVCenter</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
              <property name="indent">
               <number>-1</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,5">
               <item>
                <widget class="QLabel" name="label_15">
                 <property name="styleSheet">
                  <string notr="true">border-image: url(:/images/resources/ShopItems/hammer.png) 0 0 0 0 stretch stretch;</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="hamLabel">
                 <property name="styleSheet">
                  <string notr="true">font-size: 16px;</string>
                 </property>
                 <property name="text">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;increases multiplier by &lt;span style=&quot; font-weight:700;&quot;&gt;0.1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                 <property name="wordWrap">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_4" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,5">
               <item>
                <widget class="QLabel" name="label_16">
                 <property name="styleSheet">
                  <string notr="true">border-image: url(:/images/resources/ShopItems/pickaxe.png) 0 0 0 0 stretch stretch;</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="pickLabel">
                 <property name="styleSheet">
                  <string notr="true">font-size: 16px;</string>
                 </property>
                 <property name="text">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;increases multiplier by &lt;span style=&quot; font-weight:700;&quot;&gt;0.25&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                 <property name="wordWrap">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_5" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,5">
               <item>
                <widget class="QLabel" name="label_17">
                 <property name="styleSheet">
                  <string notr="true">border-image: url(:/images/resources/ShopItems/child.png) 0 0 0 0 stretch stretch;</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="childLabel">
                 <property name="styleSheet">
                  <string notr="true">font-size: 16px;</string>
                 </property>
                 <property name="text">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;increases multiplier by &lt;span style=&quot; font-weight:700;&quot;&gt;0.5&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                 <property name="wordWrap">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_6" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,5">
               <item>
                <widget class="QLabel" name="label_18">
                 <property name="styleSheet">
                  <string notr="true">border-image: url(:/images/resources/ShopItems/drill.png) 0 0 0 0 stretch stretch;</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="drillLabel">
                 <property name="styleSheet">
                  <string notr="true">font-size: 16px;</string>
                 </property>
                 <property name="text">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;increases multiplier by &lt;span style=&quot; font-weight:700;&quot;&gt;2.0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                 <property name="wordWrap">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_7" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,5">
               <item>
                <widget class="QLabel" name="label_19">
                 <property name="styleSheet">
                  <string notr="true">border-image: url(:/images/resources/ShopItems/dynamite.png) 0 0 0 0 stretch stretch;</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="dynLabel">
                 <property name="styleSheet">
                  <string notr="true">font-size: 16px;</string>
                 </property>
                 <property name="text">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;increases multiplier by &lt;span style=&quot; font-weight:700;&quot;&gt;5.0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                 <property name="wordWrap">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
